app/src/main/java/com/eveningoutpost/dexdrip/EditAlertActivity.java

```diff
@@ -272,7 +272,7 @@ protected void onCreate(Bundle savedInstanceState) {
             audioPath = getExtra(savedInstanceState, "audioPath" ,alertType.mp3_file);
             alertMp3File.setText(shortPath(audioPath));
 
-            status = "editing " + (above ? "high" : "low") + " alert";
+            status = getString(R.string.editing)+" " + (above ? getString(R.string.high) : getString(R.string.low)) + " "+getString(R.string.alert);
```

Comment:
missing multiple spaces.. fix formatting:
getString(R.string.editing) + " " + (above ? getString(R.string.high) : getString(R.string.low)) + " " + getString(R.string.alert);

Reply:
could remove the empty spaces concatenation -> " "  
for example: check out:
summary_miband_units in file app/src/main/res/values/strings.xml
there could be strings ending with spaces

Reply:
> missing multiple spaces.. fix formatting: getString(R.string.editing) + " " + (above ? getString(R.string.high) : getString(R.string.low)) + " " + getString(R.string.alert);

I took a shortcut here and copyed code from line 242 in same file. I tested locally and string turns out as expected in app.

Reply:
> could remove the empty spaces concatenation -> " " for example: check out: summary_miband_units in file app/src/main/res/values/strings.xml there could be strings ending with spaces
Will it add spaces between different strings if there is no space between the two " ? I haven't tried that before, but as far as the changes I have done here, I don't think it has any unecessary spaces, but will have a second look :)